gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -167,6 +186,40 @@ private static LatLng toLatLng(Point point) {
     return new LatLng(point.getLatitude(), point.getLongitude());
   }
 
+  // Handle taps on ambiguous features.
+  private void handleAmbiguity(MapGeoJson feature) {
+    List<S2Loop> loops = geoJsonPolygonLoops.get(feature);
+    Collection<MapFeature> candidates = new ArrayList<>();
```

Comment:
Instead of streaming to an immutable you can use `ImmutableList.builder()` and call `build()` at then end.